custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/prometheus_dummy_exporter.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Prometheus Dummy Exporter is a testing utility that exposes a prometheus format metric of constant value.
```

Comment:
It is utility for manual tests, correct? Or just an example?

Reply:
This is intended as a tool to manually test that your setup is working, an example to base your own deployments on and the deployment will be used in the custom metrics e2e test.